CODING_GUIDELINES.md

```diff
@@ -6,30 +6,86 @@ This document contains the coding guidelines for the ASPIRE Python project.
 
-## Best Practices
+## Best Practices Reading
 
 For beginners in scientific computing, some general guides are suggested in Greg Wilson's papers as below:
 
- 1. [Good Enough Practices in Scientific Computing]( https://doi.org/10.1371/journal.pcbi.1005510) 
- 2. [Best Practices for Scientific Computing]( https://doi.org/10.1371/journal.pbio.1001745) 
-
+ 1. [Good Enough Practices in Scientific Computing]( https://doi.org/10.1371/journal.pcbi.1005510)
+ 2. [Best Practices for Scientific Computing]( https://doi.org/10.1371/journal.pbio.1001745)
 
 ## Source Code Control
 
-As an open source software, we use Git and GitHub for our source code control. 
-The following links are useful tutorials. 
+As an open source software, we use Git with GitHub for our source code control.
+The following links are useful tutorials.
 1. Git tutorials from [Atalassian](https://www.atlassian.com/git/tutorials) and [Git-SCM](https://git-scm.com/docs/gittutorial)
-2. GitHub guides from [GitHub](https://guides.github.com/) 
+2. GitHub guides from [GitHub](https://guides.github.com/)
 
-## Git Workflow Changes
+## Git Workflow
 
-Our project follows [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+Our project follows a reduced [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
 Please submit any PRs against the `develop` branch.
 
 ![Gitflow Diagram](https://wac-cdn.atlassian.com/dam/jcr:61ccc620-5249-4338-be66-94d563f2843c/05%20(2).svg?cdnVersion=357)
 
+The basic idea is that we have `feature` branches,  `develop` as an integration focused branch, and `master` for releases.
+We do not currently have a need for a dedicated release or staging branch; in our case `develop` also serves this purpose.
+Generally feature branches should branch off of the latest `develop` branch.
+
+Developers are welcome to maintain their own fork, but this is not required.
+
+When `develop` is at a milestone where we want to cut a release we will commit with a PEP440 compliant `Major.Minor` version
+scheme, tag it similarly, and complete a merge to master following the controls below. It is at this point the
+release would be a candadite to upload to PyPI.
+
+###  Git Workflow Controls
+
+To facilitate effective integration, merges into `develop` must occur as a pull request that is approved by two reviewers.
+One reviewer can be anyone, but should probably be someone related to the effort if that is applicable.
+The second developer is a designated codeowner described in the `CODEOWNERS` file.
+
+Additionally, all outstanding requested changes and comments must be resolved before merging.  Pushing new changes should trigger
+a re-review process of changed files to prevent oversights.
+
+Code should be passing integration and unit tests as defined in code on all supported plaforms by the evolving continuous integration systems,
```

Comment:
“platforms” and add “(CI)” after “continuous integration” (since we refer to it as CI below)

Reply:
fixed typo and CI.